functions/common/methods.ts

```
@@ -410,37 +465,1072 @@ export const getThisGamesAmbassadors = async (client: BikeTagClient, adminBikeTa
+
+export const parseTagnumberFromStorageKey = parseTagnumberFromQueueKey
+
+const getMainFoundFileKey = (gameSlug: string, tagnumber: number): string =>
+  `main/${gameSlug}-tag-${tagnumber}--found.webp`
+
+const normalizePlayerName = (name?: string): string => (name ?? '').trim().toLowerCase()
+
+const getMainTagForRound = (
+  round: number,
+  main: MainFolderContext,
+): Tag | undefined => {
```

ESLint

[error] Replace `⏎··round:·number,⏎··main:·MainFolderContext,⏎` with `round:·number,·main:·MainFolderContext`

(prettier/prettier)